src/App.tsx

```diff
@@ -3,6 +3,7 @@ import { BrowserRouter as Router, Routes, Route } from 'react-router-dom'
 import Layout from './components/Layout/Layout'
 import HomePage from './pages/HomePage/HomePage'
 import CaseStudyPage from './pages/CaseStudyPage/CaseStudyPage'
+import './global.css'
```

Comment:
`import './global.css'`

Remove this line, its already there in `index.tsx` line number 3